__tests__/app.test.js

```diff
@@ -154,3 +153,72 @@ describe('GET /api/reviews/:review_id/comments', () => {
   });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+  const newComment = {
+    username: 'weegembump',
+    body: 'Not Enough DICE!!!!'
+  }
+  test('should add a comment to the database and respond wiht the newly created comment', () => {
+    return request(app)
+    .post("/api/reviews/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then(({ body }) => {
+      const { amendedComment } = body;
+      const editedComment = [{
+        comment_id: expect.any(Number),
+        body: 'Not Enough DICE!!!!',
+        review_id: 1,
+        author: 'weegembump',
+        votes: 0,
+        created_at: expect.any(String)
+      }]
+      expect(amendedComment).toEqual(editedComment);
+    })
+  });
+  test('should return 400 when provided an invalid id', () => {
+    return request(app)
+    .post("/api/reviews/banana/comments")
+    .send(newComment)
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad Request");
+    })
+  });
+  test('should return 404 when review id not found', () => {
+    return request(app)
+    .post("/api/reviews/1000/comments")
+    .send(newComment)
+    .expect(404)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe('Path Not Found');
```

Comment:
I think message "path not found" is a little misleading here - it suggests the url is not found rather than the article itself